coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -871,76 +871,72 @@
-
-            for cid in concept_ids:
-                resource = retrieve_concept_resources(tx, material_id, cid)
-                # logger.info(resource)
-                relationship = retrieve_resource_to_concept_relationships(tx, cid)
-                # logger.info(relationship)
-                resources.extend([r for r in resource if r not in resources])
-                relationships.extend([r for r in relationship if r not in relationships])
+        with self.driver.session() as session:
+            try:
+                if len(wikipedia_articles) != 0:
+                    for index, resource in wikipedia_articles.iterrows():
```

Ruff (0.8.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)